extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1423,6 +1423,39 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
       },
     },
   },
+
+  'topmailru': {
+    'transport': {
+      'beacon': false,
```

Comment:
Just to confirm with you that your server does not support `beacon` `xhrpost`? They are recommended.